src/main/java/com/edmunds/tools/databricks/maven/UpsertJobMojoNoProject.java

```diff
@@ -44,6 +44,12 @@
 
     @Override
     protected JobTemplateModel getJobTemplateModel() throws MojoExecutionException {
-        return JobTemplateModel.loadJobTemplateModelFromFile(jobTemplateModelFile);
+        JobTemplateModel serializedJobTemplate = JobTemplateModel.loadJobTemplateModelFromFile(jobTemplateModelFile);
+        //We now set properties that are based on runtime and not buildtime. Ideally this would be enforced.
```

Comment:
Is this flipped? I see the properties in the build when I think they should be runtime?

Reply:
Thought this comment describing what is happening here as the default case which is build time properties. Now I see that it is describing what is happening in the next few lines.